samples/ck-rx65n/basic-sample/src/demo_scanf.c

```diff
@@ -0,0 +1,53 @@
+#include <tx_api.h>
+
+#include "r_cg_macrodriver.h"
+#include <platform.h>
+#include <Config_SCI5.h>
+
+#include "demo_scanf.h"
+#include "demo_printf.h"
+#include "app_config.h"
+
+//
+// scanf() implementation only required for IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+//
+#ifdef IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
```

Comment:
Why only if interactive. The code would work and be useful to someone even if not running in interactive demo mode, right?

Reply:
Well, just trying to save a bit of memory -- but I've removed check, etc. as requested.

Reply:
Thanks!